docs/events/timeline-ui-overview.asciidoc

```diff
@@ -117,26 +117,27 @@ Timeline, some fields may be disabled. For more information, see
 [[timeline-to-cases-ui]]
 == Attach Timeline to a case
 
-To attach a Timeline to a new or existing case, click *Attach to case* in the upper
+To attach a Timeline to a new or existing case, open it, click *Attach to case* in the upper
 right corner, and then select one of these:
 
-* *Attach timeline to new case*
-* *Attach timeline to existing case*
+* Attach to new case
+* Attach to existing case...
 
 For more information about cases, see <<cases-overview, Cases>>.
 
 [discrete]
 [[manage-timelines-ui]]
 == Manage existing Timelines
 
-You can view, duplicate, delete, and create templates from existing Timelines:
+You can view, duplicate, export, delete, and create templates from existing Timelines:
 
-. Select the *Timelines* tab.
+. Go to *Investigate* -> *Timelines*.
 . Click the *All actions* icon in the relevant row, and then select the action:
 
 * Create template from timeline (see <<timeline-templates-ui>>)
 * Duplicate timeline
-* Delete timeline
+* Export selected (see <<import-export-timelines>>)
+* Delete selected
```

Comment:
Again, I think the names of the actions should be bold since they're UI controls the user is interacting with.